frontend/src/app/about/page.module.css

```diff
@@ -3,22 +3,34 @@
     margin: 20px;
 }
 
+.container {
```

Comment:
Could: The container have a little more left and right padding so it looks a bit better in mobile view?

Reply:
I think it looks nice with the padding as it is. Can we keep it with this? then when it's deployed to Vercel I will check it on my phone and see how it looks?